### ISSUE TITLE
isExists() should just be exists()

### ISSUE THREAD

**Issue by CraigSiemens** [realm/SwiftLint#481]
`isExists()` isn't the correct way to phrase that statement and makes the code harder to read. Simply using `exists()` makes it easer to read and grammatically correct.

```swift
json["user"].exists()
```
